### PR TITLE
Add functionality to set labels as html

### PR DESCRIPTION
**Pull request by chivandikwa**
Added functionality to allow for html labels.

Related to issue: https://github.com/Rubjerg/Graphviz.NetWrapper/issues/14

### CODE REVIEW

**Comment by Chiel92** [comment]
Thanks, looks good. It seems that the CI suddenly broke with no code changes -.- I'm gonna have to fix that first before I can merge

**Comment by chivandikwa** [comment]
> Thanks, looks good. It seems that the CI suddenly broke with no code changes -.- I'm gonna have to fix that first before I can merge

Any ETA on this? I really depend on this change for my use cases of your awesome wrapper.

File: Rubjerg.Graphviz/ForeignFunctionInterface.cs
Comment (Chiel92) [inline Rubjerg.Graphviz/ForeignFunctionInterface.cs:148]: I think a name like `AddLabel` would be clearer. The Ag prefix is only taken over from functions that are more or less directly exposed (modulo the mutex). And this function would work for any object, not just nodes, correct?

**Comment by Chiel92** [comment]
> > Thanks, looks good. It seems that the CI suddenly broke with no code changes -.- I'm gonna have to fix that first before I can merge
> 
> Any ETA on this? I really depend on this change for my use cases of your awesome wrapper.

The CI doesn't seem to have a trivial fix, but since you are only exposing new functionality, nothing existing can break, we don't have to wait till I fix the CI.

**Comment by Chiel92** [comment]
Ok I fixed appveyor. If you merge master into your branch the build should be green.

File: Rubjerg.Graphviz/ForeignFunctionInterface.cs
Comment (chivandikwa) [reply Rubjerg.Graphviz/ForeignFunctionInterface.cs:176]: Yes, you are right. Agreed and updated

**Comment by chivandikwa** [comment]
> Ok I fixed appveyor. If you merge master into your branch the build should be green.

My apologies for going silent, somehow missed these notifications.

I have merge as instructed and addressed one change request.

File: Rubjerg.Graphviz/ForeignFunctionInterface.cs
Comment (Chiel92) [inline Rubjerg.Graphviz/ForeignFunctionInterface.cs:157]: This method seems obsolete to me, as it does the same `AddLabel`, correct?

**Review by Chiel92 — CHANGES_REQUESTED** [review]
Thanks for your update @chivandikwa. I've got another small comment in response.

File: Rubjerg.Graphviz/ForeignFunctionInterface.cs
Comment (Chiel92) [inline Rubjerg.Graphviz/ForeignFunctionInterface.cs:161]: This should still be removed in favour of the member variable `LabelKey`

**Review by Chiel92 — APPROVED** [review]

**Pull request merged**
